homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
I added that because it had been in an earlier build and removed, which left no brightness slider at all.

@Teagan42 approved the change. I'm just a hack. Perhaps she can shed light on how this should be. ;)